contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -2149,6 +2150,139 @@ search_partition(PG_FUNCTION_ARGS)
 	PG_RETURN_INT32(result);
 }
 
+/*
+ * Structure to cache metadata needed in isnumeric().
+ */
+typedef struct IsNumericIOData
```

Comment:
nit: maybe shift it to a header file?

Reply:
No need to shift it to header file as we are using this struct in this file only.